main.tf

```diff
@@ -0,0 +1,34 @@
+locals {
+  namespace_name                     = "${var.product}-${var.env}"
+  resource_group_name                = "${var.product}-${var.env}"
+}
+
+resource "azurerm_resource_group" "servicebus_resource_group" {
+  name     = "${local.resource_group_name}"
+  location = "${var.location}"
+}
+
+module "namespace" {
+  // TODO: reference a specific release
+  source          = "git@github.com:hmcts/terraform-module-servicebus-namespace.git"
+  name            = "${local.namespace_name}"
+  location        = "${var.location}"
+  resource_group_name  = "${local.resource_group_name}"
+}
+
+module "private_beta_activation_topic" {
+  // TODO: reference a specific release
+  source          = "git@github.com:hmcts/terraform-module-servicebus-topic.git"
+  name            = "reform.private-beta.account.activation"
```

Comment:
shouldn't this name have 'idam' somewhere?

Reply:
this topic is in `servicebus-idam` namespace